python/TestHarness/TestHarness.py

```diff
@@ -466,9 +463,10 @@ def normalizeStatus(self, job):
         """ Determine when to use a job status as opposed to tester status """
         tester = job.getTester()
         message = job.getStatusMessage()
-        if job.isFail():
+        if job.isTimeout():
+            tester.setStatus(tester.no_status, 'FAILED (TIMEOUT)')
+        elif job.isFail():
```

Comment:
This is probably ok for now, but the Job should actually be handling this logic, not the TestHarness.